docs/getting-started.md

```diff
@@ -17,9 +17,9 @@ Before you get started:
 
 ## Install KUDO into your cluster
 
-- `kubectl create -f https://raw.githubusercontent.com/kudobuilder/kudo/master/docs/deployment/00-prereqs.yaml`
-- `kubectl create -f https://raw.githubusercontent.com/kudobuilder/kudo/master/docs/deployment/10-crds.yaml`
-- `kubectl create -f https://raw.githubusercontent.com/kudobuilder/kudo/master/docs/deployment/20-deployment.yaml`
+- `kubectl create -f https://raw.githubusercontent.com/kudobuilder/kudo/b49ff7d48547bff8c0d2a5fbc01d9ffbae49adbd/docs/deployment/00-prereqs.yaml`
```

Comment:
Can we use a release tag instead?

Reply:
yeah I had the same thought, but in 0.2.0 these files did not exist :(

Reply:
dito

Reply:
I would be in favour of creating a tag especially as this is the getting started guide which will be the first exposure when people try kudo